google-datacatalog-looker-connector/tests/google/datacatalog_connectors/looker/prepare/datacatalog_tag_factory_test.py

```diff
@@ -397,8 +401,9 @@ def test_make_tag_for_query_no_model_connection_should_succeed(self):
         }
 
         assembled_metadata = entities.AssembledQueryMetadata(
-            serialize.deserialize(json.dumps(query_data), models.Query),
-            'select *', None, None)
+            serialize.deserialize31(data=json.dumps(query_data),
```

Comment:
WDYT about trying to manually format this statement to avoid such an odd *yapf* formatting?